core/metadata_db.h

```diff
@@ -24,6 +24,14 @@
  *  e-mail address 'xmipp@cnb.csic.es'
  ***************************************************************************/
 
+/* This file contains (old) database implementation of MetaData.
+ * MetaData are stored in SQL database.
+ * Some databse-specific commands are available in this implementation only.
+ * NOTES:
+ *  1) It's quite fast to iterate over ids.
+ *  2) It's slow to iterate over rows.
+ */
```

Comment:
Please add also this link: https://github.com/I2PC/xmipp-portal/wiki/MetaData---SQL-best-practices